diff -ru rxvt-unicode-9.22/src/command.C rxvt-unicode-9.22-patched/src/command.C
--- rxvt-unicode-9.22/src/command.C	2016-01-18 14:35:08.000000000 -0500
+++ rxvt-unicode-9.22-patched/src/command.C	2017-02-03 07:46:03.863055987 -0500
@@ -788,19 +788,23 @@
             }
         }
 
-      if (ctrl && meta && (keysym == XK_c || keysym == XK_v))
+      if (ctrl && (keysym == XK_c || keysym == XK_v))
         {
           if (keysym == XK_v)
-            selection_request (ev.time, Sel_Clipboard);
-          else if (selection.len > 0)
+            {
+              selection_request (ev.time, Sel_Clipboard);
+              return;
+            }
+          else if (selection.len > 0 &&
+              (selection.beg.row != selection.end.row
+               || selection.beg.col != selection.end.col))
             {
               free (selection.clip_text);
               selection.clip_text = rxvt_wcsdup (selection.text, selection.len);
               selection.clip_len = selection.len;
               selection_grab (CurrentTime, true);
+              return;
             }
-
-          return;
         }
 
 #if ENABLE_FRILLS || ISO_14755
